packages/core/src/shared/indexNode.ts

```diff
@@ -0,0 +1,13 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * Contains exports that only work for node, NOT web.
+ * Attempting to import from this file in web will throw an error,
+ * likely `TypeError: Cannot read properties of undefined (reading 'native')`
+ */
+
+export * as filetypes from './filetypes'
```

Comment:
what is node-only about filetypes.ts ? we should try to avoid exposing node-only or web-only in our index.ts files. that's an implementation detail

Reply:
Something imported in `SchemaService` is node only and I haven't looked into what it was yet. `filetypes` uses `SchemaService`, and adding the schema service to the standard `index.ts` caused web to fail even if it doesn't use that import.
I'm not sure what the alternative is besides finding and eliminating the node dependency in `SchemaService`